mender-update/deployments.cpp

```diff
@@ -314,6 +315,93 @@ error::Error PushStatus(
 		});
 }
 
+static const string logs_uri_suffix = "/log";
+
+error::Error PushLogs(
+	const string &deployment_id,
+	LogMessages messages,
```

Comment:
One thing which only occurred to me now, is that messages are stored in JSON format in the deployment log files, aka [MEN-6578](https://northerntech.atlassian.net/browse/MEN-6578), which you are now working on. And some logs can get pretty big, so I wonder if this should take a reader instead of a vector, and that you can simply assume that the content is line-by-line valid JSON (and if it's not, you could insert the broken line into its own message, perhaps, with the same timestamp as the previous valid line; useful if the device crashed).

Sorry, I should have thought of this sooner.

Reply:
I also updated https://northerntech.atlassian.net/browse/MEN-6578.